Failed to parse source map of react-double-scrollbar

**Describe the bug**
This is not a material-table-core problem itself, but a rather from the dependecy react-double-scrollbar. This was already reported in rds repository [(https://github.com/umchee/react-double-scrollbar/issues/9)](https://github.com/umchee/react-double-scrollbar/issues/9) , and seems to be introduced on the release that material-table-core is using. 

The project react-double-scrollbar seems to be not very active, so maybe a version downgrade of this dependecy should be performed.


**To Reproduce**
Just install the latest version of material-table-core and start application. 

**Additional context**
A workaround is to disable the source map generation or add a resolution in case of using package.json.

@donavan-tangible Thank you for forking this repo to help solve this issue! The above addition to my `package.json` does successfully pull from your fork, but doesn't resolve the issue because your `master` branch is identical to umchee's. When I use `"IniterWorker/react-double-scrollbar#a59d281"` to pull from your `package-map-missing-issue` branch, my compile fails to resolve:

```
Failed to compile.

Module not found: Error: Can't resolve 'react-double-scrollbar' in '[...]/node_modules/@material-table/core/dist/components/MTableScrollbar'
ERROR in ./node_modules/@material-table/core/dist/components/MTableScrollbar/index.js 10:51-84
Module not found: Error: Can't resolve 'react-double-scrollbar' in '[...]/node_modules/@material-table/core/dist/components/MTableScrollbar'

webpack compiled with 1 error
```

In this case, my `node_modules/react-double-column` is just a copy of the branch without the `dist` or its contents.

> @donavan-tangible Thank you for forking this repo to help solve this issue! The above addition to my `package.json` does successfully pull from your fork, but doesn't resolve the issue because your `master` branch is identical to umchee's. When I use `"IniterWorker/react-double-scrollbar#a59d281"` to pull from your `package-map-missing-issue` branch, my compile fails to resolve:
> 
> ```
> Failed to compile.
> 
> Module not found: Error: Can't resolve 'react-double-scrollbar' in '[...]/node_modules/@material-table/core/dist/components/MTableScrollbar'
> ERROR in ./node_modules/@material-table/core/dist/components/MTableScrollbar/index.js 10:51-84
> Module not found: Error: Can't resolve 'react-double-scrollbar' in '[...]/node_modules/@material-table/core/dist/components/MTableScrollbar'
> 
> webpack compiled with 1 error
> ```
> 
> In this case, my `node_modules/react-double-column` is just a copy of the branch without the `dist` or its contents.

Hey there 👋🏼 😄 

what is the solution until #804 is merged?

were you able to solve the issue?

--- a/package.json
+++ b/package.json
@@ -121,7 +121,6 @@
     "deep-eql": "^4.1.1",
     "deepmerge": "^4.2.2",
     "prop-types": "^15.8.1",
-    "react-double-scrollbar": "0.0.15",
     "uuid": "^9.0.0",
     "zustand": "^4.3.0"
   },

--- a/src/components/MTableScrollbar/index.js
+++ b/src/components/MTableScrollbar/index.js
@@ -1,10 +1,13 @@
 import React from 'react';
 import { Box } from '@mui/material';
-import DoubleScrollbar from 'react-double-scrollbar';
 
-const horizontalScrollContainer = {
+const doubleStyle = {
   overflowX: 'auto',
-  position: 'relative',
+  position: 'relative'
+};
+
+const singleStyle = {
+  ...doubleStyle,
   '& ::-webkit-scrollbar': {
     WebkitAppearance: 'none'
   },
@@ -12,18 +15,14 @@ const horizontalScrollContainer = {
     height: 8
   },
   '& ::-webkit-scrollbar-thumb': {
-    borderRadius: 4,
+    backgroundColor: 'rgba(0, 0, 0, .3)',
     border: '2px solid white',
-    backgroundColor: 'rgba(0, 0, 0, .3)'
+    borderRadius: 4
   }
 };
 
 const ScrollBar = ({ double, children }) => {
-  if (double) {
-    return <DoubleScrollbar>{children}</DoubleScrollbar>;
-  } else {
-    return <Box sx={horizontalScrollContainer}>{children}</Box>;
-  }
+  return <Box sx={double ? doubleStyle : singleStyle}>{children}</Box>;
 };
 
 export default ScrollBar;

